assets/app.js

```diff
@@ -437,6 +437,13 @@ new Vue({
                 return this.settings.supportedLanguages
             }
         },
+        selectedDirtyProducts: function () {
+            list_products = this.products.filter(function (product) {
```

Comment:
I'd just return directly without adding it to list_product, or are we using list_products somewhere? since it's not using, var, let or const?